### PR TITLE
Handle clicks in Link ID With Patient sheet

### PR DESCRIPTION
**Pull request by vinaysshenoy**
Pivotal Story Link: https://www.pivotaltracker.com/story/show/165407904

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientSheet.kt [hunk 1]
```diff
@@ -31,12 +32,15 @@ class LinkIdWithPatientSheet : BottomSheetActivity() {
   companion object {
     private const val KEY_PATIENT_UUID = "patientUuid"
     private const val KEY_IDENTIFIER = "identifier"
+    private const val EXTRA_WAS_ID_LINKED = "wasIdLinked"
```
[the remark this answers is not in the document]
Comment (vinaysshenoy) [reply app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientSheet.kt]: Makes sense. I'll do this.

File: app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt [hunk 2]
```diff
@@ -386,7 +386,7 @@ class PatientSummaryScreenControllerTest {
     uiEvents.onNext(PatientSummaryAllBloodPressuresDeleted(false))
     uiEvents.onNext(PatientSummaryBackClicked())
 
-    verify(screen, never()).goBackToPatientSearch()
+    verify(screen, never()).goBack()
```
[the remark this answers is not in the document]
Comment (vinaysshenoy) [reply app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt]: Done.

**Comment by vinaysshenoy** [comment]
@sanchita I needed to change `LinkIdWithPatientSheet` to a view instead of an Activity and place it in the `PatientSummaryScreen` layout.

**Comment by sanchita** [comment]
Reviewed and left comments for this PR on https://github.com/simpledotorg/simple-android/pull/614
Please address them here and close the comments there.

**Pull request merged**